client/src/app/app.component.css

```diff
@@ -0,0 +1,57 @@
+.dashboard {
+  padding-left: 150px;
+}
+
+.dashboard hr {
+  margin-top: 1.5rem;
+  margin-bottom: 1rem;
+  margin-left: 5rem;
+  margin-right: 3rem;
+  border: 0;
+  border-top: 0.5px solid rgba(248, 245, 245, 0.377);
+}
+
+.h1 {
+  padding-left: 80px;
+  padding-top: 30px;
+  font-size: 40px;
+}
+
+.sidebar {
+  height: 100%;
+  width: 200px;
+  margin-top: -113px;
+  background-color: #260859;
+  padding-top: 30px;
+  /* border-radius: 20px; */
+}
+
+.sidebar a {
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 6px 16px 16px;
+  text-decoration: none;
+  font-size: 25px;
+  color: #8f8e8e;
+  display: block;
+}
+
+.sidebar h2 {
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Is there a way that you can set `font-family` at the `window/document` level so you don't repeat yourself?